templates/patterns/popover/popover.ui_patterns.yml

```diff
@@ -0,0 +1,63 @@
+popover:
+  label: "Popover"
+  description: "The Popover JavaScript plugin is used to show and hide content. Buttons are used as triggers that are mapped to specific elements you toggle. https://v5.getbootstrap.com/docs/5.0/components/popover/"
+  variants:
```

Comment:
In my tests this always displays as link now, and the variants are primary, secondary, so on, these are not applied anymore, remove this